server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -362,6 +363,7 @@ public void apply(Settings value, Settings current, Settings previous) {
                 NetworkModule.TRANSPORT_SSL_DUAL_MODE_ENABLED,
                 NetworkModule.TRANSPORT_SSL_ENFORCE_HOSTNAME_VERIFICATION,
                 NetworkModule.TRANSPORT_SSL_ENFORCE_HOSTNAME_VERIFICATION_RESOLVE_HOST_NAME,
+                NetworkPlugin.AuxTransport.AUX_TRANSPORT_TYPES_SETTING,
```

Comment:
Missing?

```suggestion
                NetworkPlugin.AuxTransport.AUX_PORT_DEFAULTS,
                NetworkPlugin.AuxTransport.AUX_TRANSPORT_PORTS,
```

Reply:
I neglected to make AUX_PORT_DEFAULTS a setting but it definitely should be.

Reply:
For some reasons thought that  AUX_PORT_DEFAULTS is fallback settings, surely we don't need it here, it is just value  indeed, thanks!